ttax/ops_test.py

```diff
@@ -329,6 +329,11 @@ def testMultiplyBatchByScalar(self):
     self.assertAllClose(res_actual2, res_desired, rtol=1e-4)  
     self.assertAllClose(res_actual3, res_desired, rtol=1e-4)  
 
+  def testNorm(self):
+    rng = jax.random.PRNGKey(0)
+    shape = (6, 6, 6, 6)
+    tt = random_.tensor(rng, shape)
+    self.assertAllClose(ops.norm(tt), ops.norm(tt, True))
```

Comment:
Would be nice to also check if it coincides with `np.linalg.norm(ops.full(tt))` but up to you if you want to do it right now